### PR TITLE
chore: run black on all files

### PR DESCRIPTION
**Pull request by busunkim96**
The result of `black .` run from the root of the repository.

Black had trouble with one file `composer/data/python2_script.py` because of a Python 2 style print statement. I manually changed it (also noted below).

### CODE REVIEW

File: composer/data/python2_script.py [hunk 1]
```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-print 'Output from Python 2.'  # noqa
+print("Output from Python 2.")  # noqa
```
Comment (busunkim96) [inline composer/data/python2_script.py]: The only manual change in this PR.

Comment (gguuss) [reply composer/data/python2_script.py]: I believe we prefer single-quotes to double-quotes.

**Comment by busunkim96** [comment]
Would it be possible to revisit the standards in the authoring guide?

Black doesn't allow for the single/double quotes to be configured (see https://github.com/psf/black#strings). I *think* it will make it easier in the long run for folks to be able to run a code formatter to achieve a consistent style.

**Comment by gguuss** [comment]
> Would it be possible to revisit the standards in the authoring guide?
> 
> Black doesn't allow for the single/double quotes to be configured (see [psf/black#strings](https://github.com/psf/black#strings)). I _think_ it will make it easier in the long run for folks to be able to run a code formatter to achieve a consistent style.

K.

[YAPF](https://github.com/google/yapf/) is another code formatter that teams at Google use and that is recommended in the [Google Python style guide](https://google.github.io/styleguide/pyguide.html).

**Comment by kurtisvg** [comment]
> > Would it be possible to revisit the standards in the authoring guide?
> > Black doesn't allow for the single/double quotes to be configured (see [psf/black#strings](https://github.com/psf/black#strings)). I _think_ it will make it easier in the long run for folks to be able to run a code formatter to achieve a consistent style.
> 
> K.
> 
> [YAPF](https://github.com/google/yapf/) is another code formatter that teams at Google use and that is recommended in the [Google Python style guide](https://google.github.io/styleguide/pyguide.html).

I think we defaulted to black because it is what the client libraries team has already established using. Perhaps we need to resist this in the next meeting.

**Comment by busunkim96** [comment]
:+1: It sounds like this needs additional discussion. I'll leave this open for now.

**Comment by busunkim96** [comment]
Closing as we've decided that black will be an optional tool contributors can use to format their code.

**Pull request closed without merging**